lib/rules/no-shadow-route-definition.js

```diff
@@ -12,6 +12,12 @@ function buildErrorMessage(opts) {
   )}`;
 }
 
+function buildUnsupportedArgumentsErrorMessage(routeInfo) {
+  return `Unsupported arguments at ${buildSourceLocationString(
```

Comment:
Instead of crashing hard for other unknown dynamic usages, we log an error instead.

Reply:
I'd prefer to silently ignore any unexpected arguments / values in route definitions. I don't think it's the job of this rule to validate that the route definitions are formed correctly. The only job of this rule is to looked for shadowed definitions. If something unexpected is encountered, we can just bail out and ignore it.

Reply:
Sure, will remove.